masterlist.yaml

```diff
@@ -314,6 +314,10 @@ common:
     name: 'skse/plugins/po3_SpellPerkItemDistributor.dll'
     display: '[Spell Perk Item Distributor](https://www.nexusmods.com/skyrimspecialedition/mods/36869/)'
 
+  - &DAR
+    name: 'skse/plugins/DynamicAnimationReplacer.dll'
+    display: '[Dynamic Animation Replacer](https://www.nexusmods.com/skyrimspecialedition/mods/33746)'
```

Comment:
We add trailing slashes `/` at the end of links, so please add it after the nexus page number: `33746/`